### ISSUE TITLE
Jupyter integration

### ISSUE THREAD

**Issue by turicas** [turicas/rows#115]

**Comment by JoaoFelipe** [comment]
Using just features that already exists at rows, it is easy to present tables in jupyter:
It is just necessary to add a _repr_html_ that calls export_to_html into Table:

```python
def _repr_html_(self):
    return rows.export_to_html(self).decode("utf-8")
rows.Table._repr_html_ = _repr_html_
```

However, it would show all the rows. Maybe it is better to add an option to export_to_html to hide some rows (e.g. Pandas shows the first 30 rows and the last 30 rows)

**Comment by ArmlessJohn404** [comment]
Hey, I used your snipped and created a Jupyter plugin
https://github.com/turicas/rows/pull/211
The slicing of the table is far from elegant but it works for now.

**Comment by turicas** [comment]
Implemented on fa0a7523c117d43c93a7cd20662c995a3207de43. :)
